### DISCUSSION TITLE
Add chain if metamask dont have

### DISCUSSION THREAD

**Discussion by callmejm** [wevm/wagmi#1905] [General]
Hi, how to trigger add chain to MetaMask when try to connect wallet? Let say my MetaMask dont have Arbitrum One, when I try to connect Arbitrum, I add Arbitrum chain first then proceed to connect

**Comment by SaadNiaziDev** [comment]
``
import{arbitrum} from 'wagmi/chains'
``
``
const {switchNetwork}=useSwitchNetwork()
``

``
switchNetwork(42161)  || switchNetwork(arbitrum.id)  
``

**Reply by callmejm** [reply]
switchNetwork will do the add chain job ?

**Reply by SaadNiaziDev** [reply]
yes it will do this job
